src/main/java/io/confluent/connect/jdbc/TimestampIncrementingOffset.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2016 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.connect.jdbc;
+
+import java.sql.Timestamp;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TimestampIncrementingOffset {
+  private Long incrementingOffset;
```

Comment:
I think it would be nicer to have these fields as final and make this class immutable.